server/views/forms/address.pug

```diff
@@ -38,7 +38,7 @@ block content
 
   h2.pagebreak Not interested?
 
-  p Not interested in being released early with Home Detention Curfew (tagging)? Just fill in the 'Opt out' form.
+  p Not interested in being released early with Home Detention Curfew (tagging)? Just fill in the 'Opt-out' form.
```

Comment:
Design confirmed that `Opt-out` is correct format so corrected across content visible to the user in the service